templates/clusterclass-template.yaml

```diff
@@ -201,13 +201,15 @@ spec:
           controlPlane: true
     name: kubeVipPodManifest
   variables:
-  - name: sshKey
+  - metadata: {}
```

Comment:
Looks a bit strange but it's the same for other places where we have ObjectMeta. The only way to change this is to make the metadata field a pointer. I think I wouldn't do that.